hivemind/optim/adaptive.py

```diff
@@ -0,0 +1,24 @@
+from typing import Optional, Sequence
+
+from hivemind.optim.collaborative import CollaborativeOptimizer
+from hivemind import DHT, TrainingAverager
+
+
+class CollaborativeAdaptiveOptimizer(CollaborativeOptimizer):
+    """
+    Behaves exactly as CollaborativeOptimizer except:
+     - averages adaptive learning rates of an optimizer
+     - doesn't average gradients
+    :param average_opt_statistics: average optimizer statistics with corresponding names in statedict
+    :param kwargs: options for CollaborativeOptimizer
+    """
+
+    def __init__(self, average_opt_statistics: Sequence[str], **kwargs):
+        self.average_opt_statistics = average_opt_statistics
+        super().__init__(self, **kwargs)
```

Comment:
Init is usually called before setting the child attributes

Reply:
Fixed. Solution -  pass as parameter